sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Config/ServiceBusExtensionConfigProvider.cs

```diff
@@ -89,6 +89,9 @@ public void Initialize(ExtensionConfigContext context)
             // register our binding provider
             ServiceBusAttributeBindingProvider bindingProvider = new ServiceBusAttributeBindingProvider(_nameResolver, _options, _configuration, _messagingProvider);
             context.AddBindingRule<ServiceBusAttribute>().Bind(bindingProvider);
+
+            context.AddBindingRule<ServiceBusAttribute>()
```

Comment:
This should be removed.

Reply:
??

Reply:
I just haven't gotten around to it yet..